geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/iterators/AttributeIndexFilteringIteratorTest.scala

```diff
@@ -65,7 +64,7 @@ class AttributeIndexFilteringIteratorTest extends Specification with TestWithDat
       // % should return all features
       val wildCardQuery = new Query(sftName, ff.like(ff.property("name"),"%"))
       checkStrategies(wildCardQuery, AttributeIndex)
-      SelfClosingIterator(fs.getFeatures()) must haveLength(16)
+      SelfClosingIterator(fs.getFeatures(wildCardQuery)) must haveLength(16)
```

Comment:
Is the wildCardQuery restrictive?  It appears that the previous line didn't filter at all.  If all features match the wildCard, it might be worth adding another test case.